### ISSUE TITLE
Java: Adding Google Cloud Platform Libraries causes an error.

### ISSUE THREAD

**Issue by kitajima-yasuhiro-nsft** [GoogleCloudPlatform/google-cloud-eclipse#3607]
In order to use Cloud Storage operations and StackDriver Logging in a web application running on Apache Tomcat, we are trying to use the Google Cloud client library from Java.

I am trying to install "Google Cloud Platform Libraries" according to the procedure described in the official reference below, but the attached image is an exception.

https://cloud.google.com/eclipse/docs/quickstart?hl=ja
https://cloud.google.com/eclipse/docs/libraries?hl=ja

![Google Cloud Platform Libraries error](https://user-images.githubusercontent.com/61305993/75007342-1f0eca00-54b8-11ea-88e6-cad3622ecc15.PNG)

```
Could not resolve libraries
Could not resolve container path: com.google.cloud.tools.eclipse.appengine.libraries/master-container
Could not resolve artifact androidx.annotation:annotation:jar:1.1.0
Missing androidx.annotation:annotation:jar:1.1.0
```

I tried the following operations and found that the same error occurred.

(1) Install "Google Cloud Tools" on the Marketplace with Eclipse
↓
(2) Specify "GOOGLE_APPLICATION_CREDENTIALS" in the Windows environment variable, and set the absolute path of a valid JSON file in the value. Perform OS restart.
↓
(3)
After that, a new project of App Engine flexible environment was created.
↓
(4) Quit the wizard without selecting anything in the "Google Cloud Platform Libraries" setting in the new creation wizard, and confirm that the project was played without error.
↓
(5) Right-click the project and select "Build Path"> "Add Library"
↓
(6) Select "Google Cloud Platform Libraries"
↓
(7) Select "Google Cloud Storage" and "Google Stack Driver Logging" and click the OK button.

The environmental information is as follows.
・ Windows10 64bit
・ Google Cloud Tools for Eclipse 1.8.3
・ Eclipse 2018-09
      * A similar error occurred when I tried it on 2019-12.
        The same error has occurred in normal Eclipse and Pleiades.

**Comment by elharo** [comment]
This is a known issue and has been fixed at head. Release probably early next week.

**Comment by shiregator** [comment]
Is the issue #3605 ?   And I suppose it will be available in 1.8.4?  Any idea when it will be available in the Eclipse Marketplace?  Thx!

**Comment by chanseokoh** [comment]
@shiregator yes, this is fixed by #3605. We aim to release 1.8.4 this week.

**Comment by kitajima-yasuhiro-nsft** [comment]
Hello.

Thank you for your quick response.

We are waiting for the release of 1.8.4.

I'm really thankful to you!
